src/releng_docs/services_shipit.rst

```diff
@@ -57,3 +60,44 @@ It does the following tasks on every run:
 
 
 .. _libmozdata: https://github.com/mozilla/libmozdata/
```

Comment:
Maybe? I assumed it was associated with shipit_bot_uplift. Maybe @La0 can tell us for sure?